lib/modules/manager/nuget/util.spec.ts

```diff
@@ -106,6 +106,156 @@ describe('modules/manager/nuget/util', () => {
         'NuGet.Common',
       ]);
     });
+
+    it('reads nuget config file with default registry disabled', async () => {
+      fs.findUpLocal.mockReturnValue(
+        Promise.resolve<string | null>('NuGet.config'),
+      );
+      fs.readLocalFile.mockResolvedValueOnce(
+        codeBlock`
+          <configuration>
+            <disabledPackageSources>
+              <add key="nuget.org" value="true" />
+            </disabledPackageSources>
+          </configuration>`,
+      );
+
+      const registries = await getConfiguredRegistries('NuGet.config');
+      expect(registries?.length).toBe(0);
+    });
+
+    it('reads nuget config file with default registry disabled and added sources', async () => {
+      fs.findUpLocal.mockReturnValue(
+        Promise.resolve<string | null>('NuGet.config'),
+      );
+      fs.readLocalFile.mockResolvedValueOnce(
+        codeBlock`
+          <configuration>
+            <packageSources>
+              <add key="contoso.com" value="https://contoso.com/packages/"/>
+            </packageSources>
+            <disabledPackageSources>
+              <add key="nuget.org" value="true" />
+            </disabledPackageSources>
+          </configuration>`,
+      );
+
+      const registries = await getConfiguredRegistries('NuGet.config');
+      expect(registries?.length).toBe(1);
+      expect(registries![0].name).toBe('contoso.com');
+      expect(registries![0].url).toBe('https://contoso.com/packages/');
+    });
```

Comment:
use single toEqual

Reply:
Sorry, TypeScript newbie 😞  . I can't find any "single" method in Typescript doc or in Jest, thus I don't understand what you mean.

Can you show me the entire line modified please?

Reply:
@viceice Let me know if the latest changes correspond to what you're expecting (pun intended).